notifications_utils/pdf.py

```diff
@@ -1,17 +1,16 @@
-import PyPDF2
-from PyPDF2 import PdfWriter
-from PyPDF2.errors import PdfReadError
+from pypdf import PdfWriter
```

Comment:
```suggestion
from pypdf import PdfWriter, PdfReader
```
You could just also import PdfReader here since it's the only other method being used here.